authorization/src/main/kotlin/com/ritense/authorization/autoconfigure/AuthorizationAutoConfiguration.kt

```diff
@@ -67,4 +72,9 @@ class AuthorizationAutoConfiguration {
     fun permissionConditionTypeModule(): Module {
         return PermissionConditionTypeModule()
     }
+
+    @PostConstruct
+    fun postConstruct() {
+        UserManagementServiceHolder(userManagementService)
```

Comment:
This could also be done in an `init {}` block, which doesn't rely on the Spring lifecycle.

Reply:
changed